components/HomeScreen.jsx

```diff
@@ -3,11 +3,11 @@ import Articles from './sub-components/Articles';
 import NavBar from "./sub-components/NavBar";
 import Header from "./sub-components/Header";
 
-const HomeScreen = ({setVotes, loading, setLoading}) => {
+const HomeScreen = ({setVotes, isLoading, setIsLoading}) => {
     return <div className={styles.grid_container}>
         <div className={styles.header}><Header /></div>
         <div className={styles.navbar}><NavBar /></div>
-        <div className={styles.articles}><Articles setVotes={setVotes} loading={loading} setLoading={setLoading}/></div>
+        <div className={styles.articles}><Articles setVotes={setVotes} isLoading={isLoading} setIsLoading={setIsLoading}/></div>
```

Comment:
Your components are already rendering box type elements, there is no need to put them in a div. Especially as semantically a div means nothing.